src/HyperFormula.ts

```diff
@@ -2680,14 +2729,20 @@ export class HyperFormula implements TypedEmitter {
    * @category Helpers
    */
   public simpleCellAddressToString(simpleCellAddress: SimpleCellAddress, sheetId: number): Maybe<string> {
+    if(!isSimpleCellAddress(simpleCellAddress)) {
+      throw new ExpectedValueOfTypeError('SimpleCellAddress', 'simpleCellAddress')
+    }
     validateArgToType(sheetId, 'number', 'sheetId')
     return simpleCellAddressToString(this.sheetMapping.fetchDisplayName, simpleCellAddress, sheetId)
   }
 
   /**
    * Returns all addresses and ranges whose computation depends on input address or range provided.
    *
-   * @param {SimpleCellAddress | AbsoluteCellRange} address - object representation of an absolute address or range of addresses
+   * @param {SimpleCellAddress | SimpleCellRange} address - object representation of an absolute address or range of addresses
+   *
+   * @throws [[ExpectedValueOfTypeError]] if address is of wrong type
```

Comment:
We may be more precise, `is not [[SimpleCellAddress]] or [[SimpleCellRange]]`
We should check if linking to those types works